### PR TITLE
Add RC test app e2e config file

### PR DESCRIPTION
**Pull request by qdpham13**
- Add config file to include the RC test app into the github actions CI workflow
- google-services.json has been added as an env var under `REMOTE_CONFIG_E2E_GOOGLE_SERVICES`

### CODE REVIEW

File: .github/workflows/config-e2e.yml [hunk 1]
```diff
@@ -0,0 +1,42 @@
+name: Firebase Remote Config Rollouts E2E Tests
+
+on:
+  schedule:
+    - cron: 24 */4 * * *  # every 4 hours @ 24 minutes past the hour
```
Comment (danasilver) [inline .github/workflows/config-e2e.yml:5]: Any reason for 24 minutes?

File: .github/workflows/config-e2e.yml [hunk 2]
```diff
@@ -0,0 +1,42 @@
+name: Firebase Remote Config Rollouts E2E Tests
```
Comment (danasilver) [inline .github/workflows/config-e2e.yml:1]: Nit: consider generalizing to just "Firebase Remote Config E2E Tests" -- though I think this is changeable later

**Review by danasilver — APPROVED** [review]
Couple questions, super minor

File: .github/workflows/config-e2e.yml (hunk 2 above)
Comment (qdpham13) [reply .github/workflows/config-e2e.yml]: Sure. I included the Rollouts b/c I wasn't sure if we plan to use this in the future as an RC test app and the project has `Rollouts` in the name. But agreed we can change this later if we want.

File: .github/workflows/config-e2e.yml (hunk 1 above)
Comment (qdpham13) [reply .github/workflows/config-e2e.yml:5]: Not really, just using the same frequency that Sessions used. This can be updated as we see fit in the future.

**Pull request merged**